Update contributing.md (pre-MVP phase) (#444)
mozilla-ai/lumigator: CONTRIBUTING.md

# Contributing

Lumigator is still in early stages of development and as such will have
large variance between even PATCH versions in terms of feature development.

> We want to encourage both suggestions and contributions from the community in the near future. Before submitting a PR, please open a discussion in a related issue.

Reviewer: I would remove "in the near future". I don't think we have a crowd looking at our repo so if someone comes and wants to contribute we will find the way to make that happen (if the contribution makes sense)

Author: Removed!

Currently, we are able to accept:

* PRs for [**bugfixes**](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+) and **documentation** only. Please include as much information as possible;
* [Suggestions](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) on making Lumigator easier to understand and contribute to.

# Where to go

* [Lumigator documentation](https://mozilla-ai.github.io/lumigator/): Describes all the components and includes usage guides.
* [Open issues](https://github.com/mozilla-ai/lumigator/issues): Browse the open issues to check if the topic you want to raise is already being discussed.

> * [Contributing](https://github.com/mozilla-ai/lumigator/blob/19b9adefe4a78c42b784461d92b92838b9662564/CONTRIBUTING.md) options.

Reviewer 2: @ividal are we changing this to:

```suggestion
[Contribute](https://github.com/mozilla-ai/lumigator/contribute): Explore the list of "good first issues" to start contributing.
```

# Contributing

Lumigator is still in early stages of development and as such will have
large variance between even PATCH versions in terms of feature development.

> We do want encourage both suggestions and contributions from the community in the near future.

Reviewer 2: Suggestion: Put these two sentences in the same paragraph:

```suggestion
We do want encourage both suggestions and contributions from the community in the near future. Being very transparent, please understand that we cannot accept all PRs, so please first open the discussion in the issues.
```

> Being very transparent, please understand that we cannot accept all PRs, so please first open the discussion in the issues.

Reviewer 2: Suggestion: Remove this line if you accept the previous comment.

```suggestion
```

> As long as this text is here, we are able to accept:

Reviewer 2: Suggestion: Be more specific; which "text"?

> * PRs for **bugfixes** and clarification in **documentation** only;

Reviewer 2: ```suggestion
* PRs for [**bugfixes**](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+) and **documentation** only. Please include as much information as possible;
```

> * [bug reports](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+), gladly: please include as much information as possible;

Reviewer 2: Remove this line if you accept the previous comment:

```suggestion
```

> * [suggestions](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) on making Lumigator easier to understand and contribute to.

Reviewer 2: ```suggestion
* [Suggestions](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) on making Lumigator easier to understand and contribute to.
```

> # Whare to go

Reviewer 2: ```suggestion
# Where to go
```

> * [Lumigator documentation](https://mozilla-ai.github.io/lumigator/), describing all the components and including usage guides;

Reviewer 2: ```suggestion
* [Lumigator documentation](https://mozilla-ai.github.io/lumigator/): Describes all the components and includes usage guides;
```

> * [open issues](https://github.com/mozilla-ai/lumigator/issues);

Reviewer 2: ```suggestion
* [Open issues](https://github.com/mozilla-ai/lumigator/issues): Browse the open issues to check if the topic you want to raise is already being discussed.
```

> * [Contributing](https://github.com/mozilla-ai/lumigator/blob/19b9adefe4a78c42b784461d92b92838b9662564/CONTRIBUTING.md) options.

Reviewer 2: You are linking to `CONTRIBUTING.md` but the user is already in this file. I suggest the following:

```suggestion
* [Contribute](https://github.com/mozilla-ai/lumigator/contribute): Explore the list of "good first issues" to start contributing.
```
